mpcstats/benchmark/gen_vms.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+repo_root = Path(__file__).parent.parent.parent
+benchmark_dir = repo_root / 'mpcstats' / 'benchmark'
+
+import sys
+sys.path.append(f'{repo_root}/mpcstats/benchmark')
+
+import subprocess
+import os
+from protocols import all_protocols
+
+for _, program, _ in all_protocols:
+    print(f'Compiling {program}...')
+    cmd = ['make', f'-j{os.cpu_count()}', program]
+    res = subprocess.run(cmd, cwd=repo_root, capture_output=True, text=True)
+
+    if res.returncode != 0:
```

Comment:
i think we can just raise exception here
```suggestion
    if res.returncode != 0:
        raise Exception(f'--> Failed w/ return code {res.returncode}. {res.stderr}')
```

Reply:
I thought it would be good to continue compiling VMs for other protocols even if some of the protocols are failing, but I think it's reasonable to raise an exception too since all the VMs needs to be compiled anyways.